components/advanced_create_post/advanced_create_post.tsx

```diff
@@ -1323,6 +1375,78 @@ class AdvancedCreatePost extends React.PureComponent<Props, State> {
+                                <PriorityLabel priority={this.props.draft.props.priority}/>
+                                <OverlayTrigger
+                                    placement='top'
+                                    delayShow={Constants.OVERLAY_TIME_DELAY}
+                                    trigger={Constants.OVERLAY_DEFAULT_TRIGGER}
+                                    overlay={(
+                                        <Tooltip id='post-priority-picker-tooltip'>
+                                            <FormattedMessage
+                                                id={'post_priority.remove'}
+                                                defaultMessage={'Remove {priority} label'}
+                                                values={{priority: this.props.draft.props.priority}}
+                                            />
+                                        </Tooltip>
+                                    )}
+                                >
+                                    <button
+                                        type='button'
+                                        className='close'
+                                        onClick={this.handleRemovePriority}
+                                    >
+                                        <span aria-hidden='true'>{'×'}</span>
+                                        <span className='sr-only'>
+                                            <FormattedMessage
+                                                id={'post_priority.remove'}
+                                                defaultMessage={'Remove {priority} label'}
+                                                values={{priority: this.props.draft.props.priority}}
+                                            />
+                                        </span>
+                                    </button>
+                                </OverlayTrigger>
+                            </div>
+                        )
+                    )}
+                    enhanceControls={[
+                        <React.Fragment key='PostPriorityPicker'>
+                            <PostPriorityPickerOverlay
+                                priority={this.props.draft?.props?.priority}
+                                show={this.state.showPostPriorityPicker}
+                                target={this.getPostPriorityPickerRef}
+                                onApply={this.handlePostPriorityApply}
+                                onHide={this.handlePostPriorityHide}
+                                defaultHorizontalPosition='left'
+                            />
+                            <OverlayTrigger
+                                placement='top'
+                                delayShow={Constants.OVERLAY_TIME_DELAY}
+                                trigger={Constants.OVERLAY_DEFAULT_TRIGGER}
+                                overlay={this.state.showPostPriorityPicker ? <span/> : (
```

Comment:
Is a HTML element really needed for that? Or can you just return `null` instead?

Reply:
unfortunately not, the `Overlay` tries to pass props to null and throws

Reply:
then, we could use a Fragment instead ... `<></>`